tests/integration_tests/functional/test_feat_parity.py

```diff
@@ -125,19 +95,18 @@ def test_feat_parity_cpuid_mpx(vm_builder, cpu_template, microvm, guest_kernel,
     # fmt: on
 
     check_cpuid_feat_flags(
-        vm_builder,
-        cpu_template,
-        microvm,
-        guest_kernel,
-        disk,
+        vm,
         must_be_set,
         must_be_unset,
     )
 
 
-def test_feat_parity_cpuid_inst_set(
-    vm_builder, inst_set_cpu_template_ext, microvm, guest_kernel, disk
-):
+@pytest.mark.parametrize(
+    "inst_set_cpu_template",
+    set(SUPPORTED_CPU_TEMPLATES).intersection(INST_SET_TEMPLATES + ["T2"]),
```

Comment:
with this added should we remove the fixture definition of inst_set_cpu_template_ext https://github.com/firecracker-microvm/firecracker/pull/3481/files#diff-57a8deea5cf913bfbffc93f8e9dfa44c57a4e67963eb1ed00f8089f42351e7d7R30 ?

Reply:
doh!... I thought I already did.